flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/format/AbstractKuduInputFormat.java

```diff
@@ -113,10 +120,7 @@ public void close() throws IOException {
                 e.printStackTrace();
```

Comment:
O know this is unrelated, but can we change this to a log?

Reply:
sure.